src/rust-2021/IntoIterator-for-arrays.md

```diff
@@ -44,11 +44,51 @@ It does not affect any other syntax such as `for e in [1, 2, 3]`, `iter.zip([1,
 Those will start to work in *all* editions.
 
 While it's a shame that this required a small hack to avoid breakage,
-we're very happy with how this solution keeps the difference between
-the editions to an absolute minimum.
-Since the hack is only present in the older editions,
-there is no added complexity in the new edition.
+this solution keeps the difference between the editions to an absolute minimum.
 
 [25]: https://github.com/rust-lang/rust/issues/25725
 [20]: https://github.com/rust-lang/rust/pull/65819
 [22]: https://doc.rust-lang.org/book/ch05-03-method-syntax.html#wheres-the---operator
+
+## Migration
+
+A migration lint, `array_into_iter`, has been added in order to aid in automatic migration of Rust 2018 codebases to Rust 2021.
+
+In order to have `rustfix` migrate your code to be Rust 2021 Edition compatible, run:
+
+```sh
+cargo fix --edition
+```
```

Comment:
I'm wondering if it might be good to explain that this lint is already a warning on all editions?  I assume most users try to avoid warnings in their code, so they will have likely already fixed it by now (or `allow`ed it). Maybe just a brief note, something like:

The `array_into_iter` lint has already been a warning by default on all editions since the 1.41 release (with several enhancements made in 1.55). If your code is already warning free, then it should already be ready to go for Rust 2021!

...or something like that?